protoc-gen-grpc-gateway/internal/gengateway/template.go

```diff
@@ -595,13 +596,16 @@ func Register{{$svc.GetName}}{{$.RegisterFuncSuffix}}Server(ctx context.Context,
 		ctx, cancel := context.WithCancel(ctx)
 	{{- end }}
 		defer cancel()
+		var stream runtime.ServerTransportStream
```

Comment:
We can update the comment on line 580 now.

Reply:
I couldn't find any other examples, so I just removed the mention of grpc.SendHeader.